tools/Google.Cloud.Tools.ReleaseManager.IntegrationTests/ContainerCommands/IntegrationTestLibraryTest.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"):
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Tools.Common;
+using Google.Cloud.Tools.ReleaseManager.ContainerCommands;
+using System.IO;
+using Xunit;
+
+namespace Google.Cloud.Tools.ReleaseManager.IntegrationTests.ContainerCommands;
+
+[Collection(nameof(ContainerCommandFixture))]
+public class IntegrationTestLibraryTest
+{
+    private readonly ContainerCommandFixture _fixture;
+
+    public IntegrationTestLibraryTest(ContainerCommandFixture fixture) => _fixture = fixture;
+
+    [Fact]
+    public void SinglePackage()
+    {
+        var context = _fixture.CreateContext();
+        var catalog = context.CreateApiCatalog("Google.Test.V1", "Google.Test.V2");
+        var codeRepo = context.CreateCodeRepo(catalog);
+        codeRepo.AddScriptMocks();
+
+        var command = new IntegrationTestLibraryCommand();
+        var options = ContainerOptions.FromArgs(
+            $"--repo-root={codeRepo.Directory}",
+            "--library-id=Google.Test.V1");
+        Assert.Equal(0, command.Execute(options));
+
+        codeRepo.AssertExist("runintegrationtests.sh.result.txt");
```

Comment:
Perhaps extract this code into an `AssertScriptArguments(params string[] expectedArgs)` method, so this would just be `AssertScriptArguments("Google.Test.V1")`?

(I can do that refactoring afterwards if you'd like.)